docs/running-a-node/configuration-https.md

```diff
@@ -0,0 +1,87 @@
+# HTTPS Support
+The RPC server supports TLS to allow HTTPS requests, as well as optional client certificates. To enable TLS, the node must first be built with the `NANO_SECURE_RPC` cmake cache flag set to `ON`.
+
+OpenSSL must be installed. When running cmake initially, you may need to set `-DOPENSSL_ROOT_DIR` as well, depending on your system.
+
+## Configuration
+The following section in `config-rpc.toml` enables TLS:
+
+```
```

Comment:
Very minor, could add `toml` here for a prettier formatting, and `json` further below.